.travis.yml

```diff
@@ -11,19 +11,20 @@ matrix:
   allow_failures:
     - env: TOXENV=isort
 before_cache:
-  - rm -f $HOME/.cache/pip/log/debug.log
+  - pyclean $HOME/virtualenv/python2.7.9/lib
 cache:
   directories:
     - pootle/static/js/node_modules
     - pootle/assets
-    - $HOME/.cache/pip
+    - $HOME/virtualenv/python2.7.9/lib
+    - $HOME/virtualenv/python2.7/bin
```

Comment:
we are not testing upstream

any time the cache is cleared we will pull in the libs

we dont need to be rebuilding python libs on every test run - would you do that on your machine?